main_ghost.py

```diff
@@ -1,464 +1,472 @@
+                        emoji_table.append(user[1])
+
+                    if i % 20 == 19:
+                        msg = await client.get_channel(element.channel).send(msg)
+                        for emoji in emoji_table:
+                            await msg.add_reaction(emoji)
+                        msg_table.append(msg)
+                        emoji_table = []
+                        msg = ''
+                    else:
+                        msg += '\n'
+
+            if msg != '':
+                msg = await client.get_channel(element.channel).send(msg)
+                for emoji in emoji_table:
+                    await msg.add_reaction(emoji)
+                msg_table.append(msg)
+            db.add_poll(msg_table,element.purpose,element.channel,element.user_id)
+            await botspam_channel.send("A poll has been created in <#{}>!".format(element.channel))
+
+        for element in mailbox.deletecategories:
+            id = element.channel
+            category = client.get_channel(id)
+            if category != None:
+                bot_message = await message.channel.send('Please react with 👍 to confirm deletion of category `' + category.name + '`.\n\nNote: This action will irrevirsibly delete all channels contained within the specified category. Please use with discretion.')
+                await bot_message.add_reaction('👍')
+                def check(reaction, user):
+                    return user == message.author and str(reaction.emoji) == '👍'
+                try:
+                    reaction, user = await client.wait_for('reaction_add', timeout=30.0, check=check)
+                except asyncio.TimeoutError:
+                    await message.channel.send('Confirmation timed out.')
+                else:
+                    await message.channel.send('Ok, I\'ll get right on that.\n\n*This might take some time.*')
+                    for channel in category.channels:
+                        await channel.delete()
+                    await category.delete()
+                    await message.channel.send('\n:thumbsup: Channels and category deleted')
+            else:
+                await message.channel.send('Sorry, I couldn\'t find that category.')
+
+    # Delete all temporary messages after "five" seconds.
+    await asyncio.sleep(120)
+    for msg in temp_msg:
+        await msg.delete()
+
+
+# Whenever the bot regains his connection with the Discord API.
+@client.event
+async def on_ready():
+    print(' --> Logged in as')
+    print('   | > ' + client.user.name)
+    print('   | > ' + str(client.user.id))
+
+    await client.get_channel(welcome_channel).send('Beep boop! I just went online!')
+
+print(splash)
+print(' --> "' + random.choice(splashes) + '"')
+print(' --> Please wait whilst we connect to the Discord API...')
+try:
+    client.run(config.WW_TOKEN)
+except:
+    print('   | > Error logging in. Check your token is valid and you are connected to the Internet.')

```

Comment:
Because I want to use the same framework for the different bots.

Some changes still need to be made, obviously. The Ghost Bot should use the Ghost Bot's token, not the Werewolf Bot's. But I decided to at least start creating the file as it looks with the Werewolf Bot currently, so that others may use the file to start working on the Ghost Bot while I finish the Werewolf Bot.